grazer/src/main/java/edu/asu/diging/grazer/web/FileUploadController.java

```diff
@@ -0,0 +1,86 @@
+package edu.asu.diging.grazer.web;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.io.IOUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.multipart.commons.CommonsMultipartFile;
+
+import edu.asu.diging.grazer.core.domain.impl.FileImpl;
+import edu.asu.diging.grazer.core.fileupload.db.impl.FileUploadDatabaseConnection;
+
+@Controller
+public class FileUploadController {
+    
+    private final String FILE_EXTENSION = ".graphml";
+    
+    @Autowired
+    private FileUploadDatabaseConnection connection;
+    
+    @RequestMapping("/save-transformation")
+    public String uploadResources(@ModelAttribute FileImpl transformation,
```

Comment:
- uploads should happen in a POST request
- 'save-transformation' is not a very good url, you can just make a POST to /transformation/add